d2bs/kolbot/libs/common/Pather.js

```diff
@@ -1316,10 +1403,10 @@ MainLoop:
 
 					delay(1000);
 				}
-			} else if (me.area === 40 && target.course[0] === 47) { // Lut Gholein -> Sewers Level 1 (use Trapdoor)
+			} /*else if (me.area === 40 && target.course[0] === 47) { // Lut Gholein -> Sewers Level 1 (use Trapdoor); Disabled in order to use the other exit instead.
```

Comment:
Why is this disabled to use the other exit?

Reply:
Because that way it'll always use the other exit that wasn't affected by the changes. I guess it was just me being lazy, I'll add something to use the old exit.